aircraft.py

```diff
@@ -969,7 +970,7 @@ def setup(self, Nclimb, Ncruise, objective, airplane, Nmission = 1):
                 TCS([climb['x_{CG}']*climb['W_{end}'] >=
                     aircraft['x_{misc}']*aircraft['W_{misc}'] \
                     + 0.5*(aircraft.fuse['W_{fuse}']+aircraft.fuse['W_{payload}'])*aircraft.fuse['l_{fuse}'] \
-                    + (aircraft['W_{tail}']+aircraft['n_{eng}']*aircraft['W_{engsys}'])*aircraft['x_{tail}'] \
+                    + (aircraft['W_{HT}']*aircraft['x_{CG_{ht}}']) + (aircraft['W_{VT}']+aircraft['W_{cone}']+aircraft['n_{eng}']*aircraft['W_{engsys}'])*aircraft['x_{CG_{vt}}'] \
                     + (aircraft['W_{wing_system}']*(aircraft.fuse['x_{wing}']+aircraft.wing['\\Delta x_{AC_{wing}}'])) \
```

Comment:
@1ozturkbe don't think this is correct. Do we have a variable for the engine location along the fuselage? I used the VT CG as an approximation...